actors/miner/src/monies.rs

```diff
@@ -288,6 +288,11 @@ pub fn locked_reward_from_reward(reward: TokenAmount) -> (TokenAmount, &'static
     (lock_amount, &REWARD_VESTING_SPEC)
 }
 
+/// Lock amount of deposit of creating new miner
+pub fn locked_new_miner_deposit(deposit: TokenAmount) -> (TokenAmount, &'static VestSpec) {
+    (deposit, &REWARD_VESTING_SPEC)
```

Comment:
Why is the deposit a parameter here if it's just passed straight back out?

Reply:
just copy a copy code from locked_reward_from_reward. In fact, I do tend to use a certain ratio of sector collateral as new miners deposit.